examples/js/loaders/FBXLoader.js

```diff
@@ -386,6 +386,25 @@
 		texture.wrapS = valueU === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 		texture.wrapT = valueV === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 
+		if ( textureNode.properties.Scaling ) {
+
+			values = textureNode.properties.Scaling.value;
+
+			if ( typeof values === 'string' ) {
+
+				values = values.split( ',' ).map( function( number ) {
+
+					return parseFloat( number );
+
+				} );
+
+			}
```

Comment:
There is a utility function included in the loader that parses a float array. So you can replace 

```js
values = textureNode.properties.Scaling.value;
 
if ( typeof values === 'string' ) {
 
 	values = values.split( ',' ).map( function( number ) {

 	       return parseFloat( number );
 
 	} );
 
}
```

with 

    var values = parseFloatArray( textureNode.properties.Scaling.value );